src/Files.Shared/Extensions/LocalizationExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.Shared.Services;
+
+namespace Files.Shared.Extensions
+{
+    public static class LocalizationExtensions
```

Comment:
That's not supposed to be here 🤔

Reply:
Sorry, I don't understand your comment.

Reply:
I mean, we decided that it should stay in Files.Backend

Reply:
An alternative solution would be to place CommonPaths in Backend. It's more logical.